src/extensions/index.js

```diff
@@ -8,5 +8,6 @@ export default [
   require('@vue-storefront/extension-payment-backend-methods/index.js'),
   require('@vue-storefront/extension-payment-cash-on-delivery/index.js'),
   require('@vue-storefront/extension-template/index.js'),
-  require('vsf-payment-stripe/index.js')
+  require('vsf-payment-stripe/index.js'),
+  require('src/extensions/cms/index.js')
```

Comment:
@anqaka if we like to register this extension by default then we should have the `config.cms` section added to `default.json`

Reply:
If we add it to the default.json, then we should modify the installer script to populate this endpoint config as well:
https://github.com/DivanteLtd/vue-storefront/blob/cf49b4c85bb562a55e475d1d7ee33c3e36ab8dfd/core/scripts/installer.js#L371

Reply:
moved to default.json and add to installer